serializers/src/test/java/io/pravega/schemaregistry/schemas/TestSchemas.java

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.schemaregistry.schemas;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.google.protobuf.DescriptorProtos;
+import com.google.protobuf.DynamicMessage;
+import com.google.protobuf.GeneratedMessageV3;
+import io.pravega.schemaregistry.contract.data.SerializationFormat;
+import io.pravega.schemaregistry.testobjs.DerivedUser1;
+import io.pravega.schemaregistry.testobjs.DerivedUser2;
+import io.pravega.schemaregistry.testobjs.SchemaDefinitions;
+import io.pravega.schemaregistry.testobjs.User;
+import io.pravega.schemaregistry.testobjs.generated.ProtobufTest;
+import io.pravega.schemaregistry.testobjs.generated.Test1;
+import io.pravega.schemaregistry.testobjs.generated.Test2;
+import org.apache.avro.specific.SpecificRecordBase;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import static io.pravega.schemaregistry.testobjs.SchemaDefinitions.JSON_SCHEMA_STRING;
+import static org.junit.Assert.*;
+
+public class TestSchemas {
```

Comment:
To be consistent with other classes, rename to `SchemaTest`.

Reply:
I think you forgot to change this one.

Reply:
Its renamed. I think you are looking at the outdated file.